docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,231 +9,1402 @@
    },
```

Comment:
<p>Are coords not explicit?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Coords are explicit, but when you define a distribution you used dims. Or what did you mean?</p>